src/Kreta/TaskManager/Application/Organization/CreateOrganizationCommand.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * This file is part of the Kreta package.
+ *
+ * (c) Beñat Espiña <benatespina@gmail.com>
+ * (c) Gorka Laucirica <gorka.lauzirika@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types = 1);
+
+namespace Kreta\TaskManager\Application\Organization;
+
+use Kreta\SharedKernel\Domain\Model\InvalidArgumentException;
+
+class CreateOrganizationCommand
+{
+    private $id;
+    private $ownerId;
```

Comment:
I guess ownerId is auto-generated each time a owner is added to an organization, therefore it should be optional